uast/uast.go

```diff
@@ -322,7 +323,7 @@ type RuntimeImport Import
 
 type RuntimeReImport RuntimeImport
 
-//type InlineImport Import
+type InlineImport Import
```

Comment:
Please document this new type.

Reply:
As mentioned in the comment for PR, I'd like to document all types here in a separate PR.

Reply:
SGTM.